src/main/java/de/doubleslash/keeptime/common/RandomColorPicker.java

```diff
@@ -0,0 +1,74 @@
+package de.doubleslash.keeptime.common;
+
+
+
+
+import de.doubleslash.keeptime.model.Project;
+import de.doubleslash.keeptime.model.Settings;
+import javafx.beans.property.ObjectProperty;
+import javafx.collections.ObservableList;
+import javafx.scene.paint.Color;
+import org.apache.commons.lang.ObjectUtils;
+import org.apache.commons.logging.Log;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+import java.lang.reflect.Field;
+import java.sql.Struct;
+import java.util.ArrayList;
+import java.util.List;
+
+public class RandomColorPicker {
+    private static final Logger LOG = LoggerFactory.getLogger(RandomColorPicker.class);
+
+   public static Color getRandomColor() throws ClassNotFoundException, IllegalAccessException {
+
+        List<Color> colors = new ArrayList<>();
+        Class clazz = Class.forName("javafx.scene.paint.Color");
+        if (clazz != null) {
+            Field[] field = clazz.getFields();
```

Comment:
not sure if this is the best idea but `Color.class.getFields()` will be a better option to implement it

Reply:
Implemented it like u said